charts/harvester-cloud-provider/harvester-cloud-provider/0.1.300+up0.1.3/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+appVersion: v0.1.1
+description: A Helm chart for Harvester Cloud Provider
+maintainers:
+- name: harvester
+name: harvester-cloud-provider
+type: application
+version: 0.1.300+up0.1.3
```

Comment:
missing expected annotations: https://confluence.suse.com/display/EN/Feature+Charts+Helm+Requirements#FeatureChartsHelmRequirements-Annotations

How do we expect to deploy this chart? Is it a normal installable chart via Apps & Marketplace or being deployed by some other mechanism? e.g. do we need a UI for this?

If we don't plan to expose a custom UI, we need `questions.yml` defined for this as well.

Reply:
same for CSI driver

Reply:
https://confluence.suse.com/display/EN/Feature+Charts+Helm+Requirements#FeatureChartsHelmRequirements-Questions.ymlDeprecation

Questions.yml is deprecated

Reply:
In that case, is there a custom UI being created for a Harvester chart install? e.g. an issue filed under https://github.com/rancher/dashboard

Reply:
Either way, we also need annotations for both charts in the Chart.yaml